### PR TITLE
[Examples] Update pagerank examples

### PR DESCRIPTION
**Pull request by Yuzhen11**

### CODE REVIEW

File: examples/pagerank.cpp [hunk 1]
```diff
@@ -62,28 +57,28 @@ void pagerank() {
...
+    husky::load(infmt, parse_wc);
+    husky::globalize(vertex_list);
 
     // Iterative PageRank computation
     auto& prch =
         husky::ChannelFactory::create_push_combined_channel<float, husky::SumCombiner<float>>(vertex_list, vertex_list);
     int numIters = stoi(husky::Context::get_param("iters"));
     for (int iter = 0; iter < numIters; ++iter) {
-        list_execute(vertex_list, [&prch, &it](Vertex& u) {
-            if (it > 0)
```
Comment (xwkuangLouis) [inline examples/pagerank.cpp]: There is a typo here, it should be "iter" instead of "it".

Comment (Yuzhen11) [inline examples/pagerank.cpp]: Yes, you are right. The old version is buggy... I fix it in this commit.

**Comment by xwkuang5** [comment]
宇桢~
下面这段代码我标了橙色的那里其实是为什么要clear buffer啊？如果现在clear掉那么block里面剩下的数据不就丢失了吗？
void HDFSLineInputFormat::handle_next_block() {
while (true) {
if (buffer_.empty())
return;
r = helper::find_next(buffer_, 0, '\n');
if (r == boost::string_ref::npos) {
// fetch next subBlock
last_part_ += std::string(buffer_.data(), buffer_.size());
buffer_ = splitter_.fetch_block(true);
continue;
} else {
last_part_ += std::string(buffer_.substr(0, r).data(), r);
clear_buffer();
return;
}
}
}

**Comment by kygx-legend** [comment]
@xwkuang5 For this problem, please don't reply the github directly. And it is not a good place to discuss such kind of problem here.

**Comment by xwkuang5** [comment]
Sorry, I thought I send an individual email. I will delete it now.

**Pull request merged**